Core/Core/Extensions/InstIconExtensions.swift

```diff
@@ -0,0 +1,421 @@
+//
+// This file is part of Canvas.
+// Copyright (C) 2019-present  Instructure, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+//
+
+// DO NOT EDIT: this file was generated by build-icons.js
+
+import SwiftUI
+import UIKit
+
+public extension UIImage {
+    static var addLine: UIImage { UIImage(named: "addLine", in: .core, compatibleWith: nil)! }
```

Comment:
I did these as vars with getters so icons don't stay in memory permanently.
With Colors, it seems like there is a not long list, & individually they won't take up much memory.
With Fonts, it seems like they will be used a lot & so probably will be in memory anyway.
With Images, the set that appears on each screen will be very different, usually a very small subset, & they are large in memory footprint.